### ISSUE TITLE
Add documentation for the Submit button pattern - teams want to use something similar and there is no guidance

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/vets-design-system-documentation#3202]

**Comment by artsymartha68** [comment]
There is nothing in the VADS site to explain the pattern for how a Submit button works.

There is a team wanting to do something similar and they have nothing to say whether or not their concept is valid. For example, there is a filter feature. The options are displayed on a new page, then there is an Apply button which closes the page, applies the filter, and redisplays the original page. The Apply button is acting like the Submit button, but there is no existing pattern for how the Submit button works to validate against.

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/3202?planning-poker) with Zenhub @babsdenney @danbrady @LWWright7

**Comment by humancompanion-usds** [comment]
@artsymartha68 - I feel this may be too fundamental a concept to explain in a Design System. I'm not seeing any other Design System explaining how a submit button works. We now have a [search filter component](https://design.va.gov/components/search-filter) and [guidance](https://design.va.gov/templates/search-results). Does that suffice?

**Comment by artsymartha68** [comment]
I agree, and we don't need to document all the "don't do this" scenarios. Yes the new search filter and guidance suffices, imo. Closing this one out, thanks for following up!
